pnpm-lock.yaml

```diff
@@ -15932,6 +15786,7 @@ packages:
   /typedoc-plugin-resolve-crossmodule-references@0.3.3(typedoc@0.23.28):
     resolution: {integrity: sha512-ZWWBy2WR8z9a6iXYGlyB3KrpV+JDdZv1mndYU6Eh6mInrfMCrQJi3Y5K9ihMBfuaBGB//le1nEmQLgzU3IO+dw==}
     engines: {node: '>=14'}
+    deprecated: Upgrade to typedoc >= 0.24 and remove typedoc-plugin-resolve-crossmodule-references from your dependencies
```

Comment:
New deprecation for a direct dependency :eyes:

Reply:
yeah typedoc has been a bit of a nightmare tbh.  i'm tempted to switch off our api docs, as I don't think anyone is using them anyways 🤔

Reply:
https://github.com/cursorless-dev/cursorless/issues/1749